src/main/java/org/folio/inventory/dataimport/handlers/actions/CreateHoldingEventHandler.java

```diff
@@ -168,27 +175,31 @@ private void prepareEvent(DataImportEventPayload dataImportEventPayload) {
     dataImportEventPayload.setCurrentNode(dataImportEventPayload.getCurrentNode().getChildSnapshotWrappers().get(0));
   }
 
-  private void fillInstanceIdIfNeeded(DataImportEventPayload dataImportEventPayload, JsonObject holdingAsJson) {
+  private void fillInstanceIdIfNeeded(String instanceId, JsonObject holdingAsJson) {
```

Comment:
The previous version of this call allowed instance id to be filled without making an extra HTTP call later. I think we should leave room for both.
- if non-ECS, get from payload
- if ECS, get from payload then verify existence